vignettes/release-announcement.Rmd

```diff
@@ -0,0 +1,131 @@
+```{r}
+#| include: false
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're excited to announce that an initial version of the S7 package is now on CRAN.
+S7 is a new OOP system designed to be a successor to S3 and S4.
+The overarching goal of S7 is to combine the best parts of S3 and S4, so that the R community can eventually standardise on an single OOP system.
+For that, S7 is designed on top of S3, making it 100% backward compatible with existing code that uses S3.
+Additionally, it's been thoughtfully designed to work with S4, and as we learn more about the challenges of transitioning from S4 to S7 we expect to add additional features to make that transition as simple as possible.
+You can learn more about our concerns with the existing OOP systems in <https://rconsortium.github.io/OOP-WG/articles/motivation.html>.
+
+To avoid the problem of designing "yet another OOP system" for R, the design and implementation of S7 has been a team effort.
+The has been working as working group of R Consortium and including representatives from R-Core, BioConductor, Posit, and the wider R community.
```

Comment:
```suggestion
The has involved an R Consortium working group including representatives from R-Core, BioConductor, Posit, and the wider R community.
```

Or something of this nature to avoid the "working as working group" language

Reply:
yes, but I am even more concerned about the validity of an English sentence startings with  "The has ..".
Probably rather  "The team has .." ?

Reply:
The 'C' is lowercase in Bioconductor. Also, when I formed the working group, I was really reaching out to the tidyverse, rather than Posit itself. Of course, I really appreciate Posit's massive contribution and want to acknowledge it. We should also mention ROpenSci.